package/src/components/Select/Select.js

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import ReactSelect from "react-select";
+import AsyncSelect from "react-select/async";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import {
+  Control,
+  Menu,
+  MultiValue,
+  NoOptionsMessage,
+  Option,
+  Placeholder,
+  ValueContainer
+} from "./helpers";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    height: 250,
```

Comment:
Seems like this `250px` of height would cause some issues like in the docs where there's extra spacing below the select: 
![image](https://user-images.githubusercontent.com/42217/63294652-b6802f80-c27f-11e9-8a96-9769a0ad75e4.png)

Reply:
I agree that space is unnecessary, removed.